google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java

```diff
@@ -403,6 +402,12 @@ Builder setCustomerEncryption(CustomerEncryption customerEncryption) {
       return this;
     }
 
+    @Override
+    public Builder setKmsKeyName(String kmsKeyName) {
```

Comment:
Is KMS support GA?

Reply:
Not yet, I'll expect merge at Beta. I'll annotate these methods with `@GcpLaunchStage.Beta`.